src/InstallWizard.js

```diff
@@ -274,6 +352,50 @@ class InstallWizard extends Component {
     }
   }
 
+  //this is triggered when start the update processes
+  //for example click repalce button on Replace server modal
+  startUpdate = (operation, pages, theProps) => {
```

Comment:
its a little weird that this stuff is in the InstallWizard.js , I guess it allows for reusing the wizard functionality when doing a multi-step process as a result of a user choice, so its probably ok

Reply:
yeah...tried to reuse the code in InstallWizard. I will add more comments.